src/Float16.h

```diff
@@ -97,93 +72,36 @@ struct float16_t {
      */
     EXPORT static float16_t make_from_bits(uint16_t bits);
 
-    /** Get a new float16_t from a signed integer.
-     *  It is not provided as a constructor to avoid call ambiguity
-     * */
-    EXPORT static float16_t make_from_signed_int(int64_t value, RoundingMode roundingMode=RoundingMode::ToNearestTiesToEven);
     /**@}*/
 
-    /**\name Arithmetic operators
-     * These compute the result of an arithmetic operation
-     * using a particular ``roundingMode`` and return a new float16_t
-     * representing the result.
-     *
-     * Exceptions are ignored.
-     */
-    /**@{*/
-    /** add */
-    EXPORT float16_t add(float16_t rhs, RoundingMode roundingMode) const;
-    /** subtract */
-    EXPORT float16_t subtract(float16_t rhs, RoundingMode roundingMode) const;
-    /** multiply */
-    EXPORT float16_t multiply(float16_t rhs, RoundingMode roundingMode) const;
-    /** divide */
-    EXPORT float16_t divide(float16_t denominator, RoundingMode roundingMode) const;
-    /** IEEE-754 2008 5.3.1 General operations - remainder **/
-    EXPORT float16_t remainder(float16_t denominator) const;
-    /** C fmod() */
-    EXPORT float16_t mod(float16_t denominator, RoundingMode roudingMode) const;
-    /**@}*/
-
-
     /** Return a new float16_t with a negated sign bit*/
     EXPORT float16_t operator-() const;
 
-    /** \name Overloaded arithmetic operators for convenience
-     * These operators assume RoundingMode::ToNearestTiesToEven rounding
-     */
-    /**@{*/
+    /** Arithmetic operators. */
```

Comment:
Rounding mode is now unspecified? Do we want to mention this explicitly?

Reply:
Rounding mode is now round to nearest, ties to even. I'll fix the comments.